notebooks/match_with_noaa_stations.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>you can take away the logging </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>